plenum/test/client/test_client.py

```diff
@@ -13,7 +12,7 @@
 from plenum.test.helper import sendReqsToNodesAndVerifySuffReplies
 from plenum.test.test_client import genTestClient
 from plenum.test.test_node import TestNodeSet
-from plenum.common.crypto import Signer
+from raet.raeting import AutoMode
```

Comment:
Probably yes, as well as all other raet-related changes. 
There is nothing new in this line (just organize import was applied).